pifi/pifi.py

```diff
@@ -71,8 +71,9 @@ def status(nm=nm):
             print("Device is not activated")
             exit(0)
         current_connection = ApModeDevice.GetAppliedConnection(0)
-        if current_connection[0]['802-11-wireless']['mode'] == "ap":
-            print("Device is currently acting as an Access Point")
+        if 'mode' in current_connection[0]['802-11-wireless']:
```

Comment:
This doesn't look right because if you have a 'mode' key but it isn't equal to "ap" then the else condition isn't run

Reply:
Oh yes you are right. I assumed that 'mode' was appearing only when 'ap' was available. I have changed this so that 'ap' mode is checked when 'mode' is in current_connection AND 'mode'=='ap'. Considering that python is safe in the order of the conditionals.